Docs 8278 update oas3 docs pa (#265)
mulesoft/docs-api-manager: modules/ROOT/pages/create-instance-task.adoc

image::manage-exchange-end-task.png[]
+
. From *API Name*, select the API (`jsonplaceholder`).
+

> NOTE: OAS 3.0 specifications are currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager.

Reviewer: Will used this as the name: "OpenAPI Specification (OAS) 3.0" and then "OAS 3.0" and "OAS 2.0."
So I think adding "specification" is redundant, and plural is not correct?
```suggestion
NOTE: OAS 3.0 is currently not supported in API Manager. You must convert the API to RAML or OAS 2.0 to manage it using API Manager.
```
